src/Compilers/VisualBasic/Portable/Symbols/FieldSymbol.vb

```diff
@@ -326,6 +326,26 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols
             End Get
         End Property
 
+        ''' <summary>
+        ''' Is this a field of a tuple type?
+        ''' </summary>
+        Public Overridable ReadOnly Property IsTupleField() As Boolean
+            Get
+                Return False
+            End Get
+        End Property
+
+        ''' <summary>
+        ''' If this is a field of a tuple type, return corresponding underlying field from the
+        ''' tuple underlying type. Otherwise, null. In case of a malformed underlying type
+        ''' the corresponding underlying field might be missing, return null in this case too.
```

Comment:
fixed

---
In reply to: [71447153](https://github.com/dotnet/roslyn/pull/12596#discussion_r71447153) [](ancestors = 71447153)